pilot/pkg/bootstrap/server.go

```diff
@@ -22,10 +22,10 @@ import (
 
 	"code.cloudfoundry.org/copilot"
 	"github.com/davecgh/go-spew/spew"
+	multierror "github.com/hashicorp/go-multierror"
```

Comment:
This alias should not be necessary.  Is there a reason it was added?

Reply:
it's auto generated by gofmt.